app/uk/gov/hmrc/cdsreimbursementclaim/connectors/EisConnector.scala

```diff
@@ -41,9 +38,9 @@ trait EisConnector {
 
   }
 
-  def enrichHC(hc: HeaderCarrier): HeaderCarrier =
+  def enrichHC(hc: HeaderCarrier, bearerToken: String): HeaderCarrier =
```

Comment:
it would be good to make change the signature of this method so that it takes the headercarrier as an implicit curried parameter.

Reply:
looks like we don't need to pass in different tokens now so we won't need this but changing it to make the header carrier be picked up implicitly would still be a good thing to do